classes/phing/tasks/ext/SassTask.php

```diff
@@ -1047,14 +1047,26 @@ public function loadScssphp()
         return $success;
     }
 
+    public function getNewCompiler()
+    {
+        $compiler = '\\Leafo\\ScssPhp\\Compiler';
+        $obj = call_user_func_array(
+            array($compiler, '__construct'),
+            array()
+        );
+        var_dump ($obj);
+        return self::$factory;
+    }
+
     /**
      * Initialise and return an instance of the ScssPhp Compiler.
      *
      * @return Leafo\ScssPhp\Compiler
      */
     public function initialiseScssphp()
     {
-        $scss = new Leafo\ScssPhp\Compiler();
+        //$scss = new Leafo\ScssPhp\Compiler();
+        $scss = $this->getNewCompiler();
```

Comment:
Bother. I'll see what I can do about this shortly. Ultimately if it's down to ScssPhp not being PHP 7 compatible a bug fix would be better going up-stream to that project.